erizo/src/erizo/rtp/BandwidthEstimationHandler.cpp

```diff
@@ -116,6 +135,104 @@ void BandwidthEstimationHandler::read(Context *ctx, std::shared_ptr<dataPacket>
       ELOG_DEBUG("Packet not parsed %d", packet->type);
     }
   }
+
+  uint32_t now = ClockUtils::timePointToMs(clock::now());
+  RtpHeader *head = reinterpret_cast<RtpHeader*> (packet->data);
+  if (!chead->isRtcp()) {
+    // RTP PACKETS
+    uint16_t seqNum = head->getSeqNumber();
+    switch (packet->type) {
+      case VIDEO_PACKET:
+      // RTP VIDEO PACKET
+        videoRR.ssrc = head->getSSRC();
+        if (!rtpSequenceLessThan(seqNum, videoRR.last_rtp_seq_num)) {
+          if (seqNum < videoRR.last_rtp_seq_num) {
+            ELOG_WARN("VIDEO RTP WRAPPED");
+            videoRR.cycle++;
+          }
+          videoRR.last_rtp_seq_num = seqNum;
+        }
+      break;
+      case AUDIO_PACKET:
+      // RTP AUDIO PACKET
+        audioRR.ssrc = head->getSSRC();
+        if (!rtpSequenceLessThan(seqNum, audioRR.last_rtp_seq_num)) {
+          if (seqNum < audioRR.last_rtp_seq_num) {
+            ELOG_WARN("VIDEO RTP WRAPPED");
```

Comment:
Same as in video... I don't think this should be a WARN... probably we don't need to log it anyway.